pkg/broker/handler/processors/filter/processor_test.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package filter
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cloudevents/sdk-go/v2/event"
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/google/knative-gcp/pkg/broker/config"
+	handlerctx "github.com/google/knative-gcp/pkg/broker/handler/context"
+	"github.com/google/knative-gcp/pkg/broker/handler/processors"
+)
+
+func TestFilterProcessor(t *testing.T) {
```

Comment:
My preference would be to rewrite this as a table test, my guess is that the test struct would look something like:

```golang
[]struct {
    filterAttributes *map[string]string
    event ce.Event
    shouldPass bool
}
```